tools/issue-labeler/issue-labeler.sln

```diff
@@ -0,0 +1,60 @@
+﻿
```

Comment:
nittiest of nits:  We should probably change to "IssueLabeler" to follow repository conventions.